.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,2 @@
+To do:
+- [ ] render the page before merging via `python render_page.py` and adding `index.html`
```

Comment:
This would be a nice pre-commit hook.

Reply:
IDK how to ship this hook to anyone who forks the repo. Suggestions welcome!